Atlas Demo

noleplay81 opened the pull request:
Update Google Atlas to use new demo

![Screenshot_1595836408](https://user-images.githubusercontent.com/54804710/88517797-5d418400-d02a-11ea-9e15-ddf2292465d9.png)
![Simulator Screen Shot - iPhone SE (2nd generation) - 2020-07-27 at 16 52 37](https://user-images.githubusercontent.com/54804710/88517807-63376500-d02a-11ea-8e74-03fd2dbb4dad.png)

Commit:
Update Google Atlas to use new demo

add project - android / ios

felangel commented:
@noleplay81 thanks so much for taking the time to put this together. What are everyone's thoughts on including these changes as part of the `example` directly in `google_atlas`?

noleplay81 commented:
> @noleplay81 thanks so much for taking the time to put this together. What are everyone's thoughts on including these changes as part of the `example` directly in `google_atlas`?

@felangel 
Thanks for the comments. Reverted changes to Google Atlas. Added information about using Demo in the AtlasDemo's Readme file.
